allensdk/brain_observatory/behavior/session_apis/data_io/behavior_ophys_json_api.py

```diff
@@ -153,3 +153,7 @@ def get_eye_gaze_mapping_file_path(self) -> str:
         """Get h5 filepath containing eye gaze behavior of the experiment's
         subject"""
         return self.data['eye_gaze_mapping_path']
+
+    def get_event_detection_filepath(self) -> str:
+        """Get the filepath of the .h5 events file associated with an ophys experiment"""
+        return self.data['event_detection_filepath']
```

Comment:
I think this needs to be `self.data['events_file']`

This is how the LIMS strategy writing the field at least:
http://stash.corp.alleninstitute.org/projects/TECH/repos/lims/browse/app/strategies/behavior_ophys_write_nwb_strategy.rb#126

It also occurs to me that you need to add that field to the Argschema Schema for the behavior ophys write nwb module:

https://github.com/AllenInstitute/AllenSDK/blob/64dd27a0525ecc8b6237cf5b488646790e48359e/allensdk/brain_observatory/behavior/write_nwb/_schemas.py#L24

Reply:
Thank you, I have updated it.